components/app.js

```diff
@@ -57,4 +63,22 @@ class App {
     }
     $.ajax("https://sgt.lfzprototypes.com/api/grades", ajaxConfig)
   }
+  deleteGrade(id) {
+    var appConfig = {
+      type: "DELETE",
+      headers: {
+        "X-Access-Token": "jpVhjpBr"
+      },
+      success: this.handleDeleteGradeSuccess,
+      error: this.handleDeleteGradeError
+    };
+    $.ajax("https://sgt.lfzprototypes.com/api/grades/" + id, appConfig)
+    console.log(id);
```

Comment:
Please remove `console.log`s after you've verified the functionality they're testing